settings.py

```diff
@@ -2,36 +2,30 @@
 # -*- coding: utf-8 -*-
 
 from wger.settings_global import *
-from decouple import config
-import dj_database_url
-import os
 
 # Use 'DEBUG = True' to get more details for server errors
 DEBUG = True
 TEMPLATES[0]['OPTIONS']['debug'] = True
 
 ADMINS = (
-    (os.environ.get('ADMIN_NAME'), os.environ.get('ADMIN_EMAIL')),
+    ('Your name', 'your_email@example.com'),
 )
 MANAGERS = ADMINS
 
-if os.environ.get("DB") == 'postgresql':
-    DATABASES = {
+
+DATABASES = {
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
             'NAME': os.environ.get('WGER_NAME') or 'wger',
             'USER': os.environ.get('WGER_USER') or 'postgres',
             'PASSWORD': os.environ.get('WGER_PASSWORD') or '',
             'HOST': os.environ.get('WGER_HOST') or 'localhost',
-            'PORT': os.environ.get('WGER_PORT') or '',
+            'PORT': os.environ.get('WGER_PORT') or '5432',
         }
     }
 
-if os.environ.get("DB") == "heroku":
-    DATABASES = {
-        'default': dj_database_url.config(default=config('DATABASE_URL'))
-    }
-
```

Comment:
@solnsubuga doesn't _heroku_ need this block?

Reply:
@Yiga-fred please add these lines back. It used in on heroku.